src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -159,7 +159,7 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                         <button
```

Comment:
What functionality does this button hold?
Previously it retriggered the problem statement extraction but now the extraction will only be executed on a new exercise update.
=> Either this button should retrigger an update or it should be removed.

Reply:
The purpose of this button is to test the implementation of the task extraction. This is also the reason why it is only accessible for Admins.
We agreed with Stephan to add such a button for testing purposes, as this is a lot better than looking into the database (which can only be done locally)
These buttons will be removed in the future and replaced by a proper modal that shows things like the detected tasks.